core/src/databases/remote_databases/get_remote_database.rs

```diff
@@ -17,5 +17,8 @@ pub async fn get_remote_database(
             let db = SnowflakeRemoteDatabase::new(content)?;
             Ok(Box::new(db) as Box<dyn RemoteDatabase + Sync + Send>)
         }
+        _ => {
+            anyhow::bail!("{:?} is not a supported remote database provider", provider)
```

Comment:
?

Reply:
Because I'm adding a new provider, I need to handle non-snowflake providers

Reply:
bail is cool (and honestly we should use it) but we don't use it anywhere else, so for consistency can your replace it with :
```
Err(anyhow!("{:?} is not a supported remote database provider", provider))?
```

please 🙏

Reply:
also please import anyhow insead of fully qualifying 🙏

Reply:
:+1: